data_store/db/entities.py

```diff
@@ -606,7 +606,7 @@ class Submission(BaseModel):
     reporting_round_id: Mapped[GUID] = sqla.orm.mapped_column(sqla.ForeignKey("reporting_round.id"), nullable=False)
 
     submission_date = sqla.Column(sqla.DateTime(), nullable=True)
```

Comment:
Should we make this non-nullable at the same time? I've just checked prod and there are 0 records without a submission date - so this feels like a good time for us to start enforcing that?

There are a few tests that will fail because of this, but I think it should just be adding `submission_date` values for them.

Reply:
Changed submission_date to be non-nullable.